### PR TITLE
Add regression guidance

### PR DESCRIPTION
**Pull request by emceeaich**

### CODE REVIEW

File: policy/triage-bugzilla.md [hunk 1]
```diff
@@ -59,6 +59,33 @@ Also, as a release approaches, the release status of open, high priority (P1) bu
...
+* Facebook homepage does not render
+* Image elements do not appear in the DOM tree
+* CSS Grid rules are not recognized
+
+#### Is not a regression
+
+* A website dependent on an unimplented part of the CSS specification does not render as intended
+* Firefox implements a feature differently than the user expects it to
+* The reporter is requesting a new feature
+
```
Comment (sylvestre) [inline policy/triage-bugzilla.md]: * A refactor in the code

File: policy/triage-bugzilla.md [hunk 2]
```diff
@@ -59,6 +59,33 @@ Also, as a release approaches, the release status of open, high priority (P1) bu
...
+#### Handling regressions
+
+If a bug has been reported with the `regression` keyword added, or the bug has the `regression` keyword, then:
+
+* The commit which casued the regression should be found and noted in the bug
+* The bug will be reviewed at the [weekly regression triage meeting](https://wiki.mozilla.org/Platform#Weekly_Regression_Triage_Meeting)
+
+Unless a bug is resolved as invalid, wontfix, or a duplicate; or if the bug is marked as P5, it is expected that the bug will refer to the commit that caused the regression. 
+
+If the commit which caused the regression has not been found, add the `regressionrange-wanted`.
```
Comment (sylvestre) [inline policy/triage-bugzilla.md]: +keyword (at the end of the sentence) not everyone knows it is a keyword

File: policy/triage-bugzilla.md [hunk 3]
```diff
@@ -59,6 +59,33 @@ Also, as a release approaches, the release status of open, high priority (P1) bu
...
+
+* A website dependent on an unimplented part of the CSS specification does not render as intended
+* Firefox implements a feature differently than the user expects it to
+* The reporter is requesting a new feature
+
+#### Handling regressions
+
+If a bug has been reported with the `regression` keyword added, or the bug has the `regression` keyword, then:
+
+* The commit which casued the regression should be found and noted in the bug
```
Comment (sylvestre) [inline policy/triage-bugzilla.md]: s/casued/caused/

File: policy/triage-bugzilla.md [hunk 4]
```diff
@@ -59,6 +59,33 @@ Also, as a release approaches, the release status of open, high priority (P1) bu
...
+* The bug will be reviewed at the [weekly regression triage meeting](https://wiki.mozilla.org/Platform#Weekly_Regression_Triage_Meeting)
+
+Unless a bug is resolved as invalid, wontfix, or a duplicate; or if the bug is marked as P5, it is expected that the bug will refer to the commit that caused the regression. 
+
+If the commit which caused the regression has not been found, add the `regressionrange-wanted`. 
+
```
Comment (sylvestre) [inline policy/triage-bugzilla.md]: It is also expected that the status flags will be updated to match which versions are affected/unaffected.

File: policy/triage-bugzilla.md [hunk 5]
```diff
@@ -59,6 +59,33 @@ Also, as a release approaches, the release status of open, high priority (P1) bu
 
 An explanation of the release status field.
 
+### Regressions
+
+_A regression is a code change that degrades existing functionality or performance for the user._
```
Comment (marco-c) [inline policy/triage-bugzilla.md]: Is it necessary to specify `for the user`?

File: policy/triage-bugzilla.md [hunk 6]
```diff
@@ -59,6 +59,33 @@ Also, as a release approaches, the release status of open, high priority (P1) bu
...
+
+#### Is a regression
+
+* Facebook homepage does not render
+* Image elements do not appear in the DOM tree
+* CSS Grid rules are not recognized
```
[the remark this answers is not in the document]
Comment (marco-c) [reply policy/triage-bugzilla.md]: Maybe also add an example of a performance regression, e.g. `* Memory usage increased while typing in awesomebar`.

File: policy/triage-bugzilla.md (hunk 1 above)
Comment (marco-c) [inline policy/triage-bugzilla.md]: A bug that is not dependent on a code change in Firefox but from external factors (e.g. a new crash caused by a Windows update)

File: policy/triage-bugzilla.md (hunk 2 above)
Comment (marco-c) [inline policy/triage-bugzilla.md]: Do we want the `regressionrange-wanted` keyword to be used when we know there's a regression but we can't pinpoint which commit caused it?
When the bug is resolved, should the `regressionrange-wanted` keyword still be there even if we don't need a regression range anymore?

**Comment by pascalchevrel** [comment]
Is there an expiration time for a regression to be qualified as such or should an old regression turn into regular bug work if the regression range is too old (too old needing to be defined)?

**Comment by marco-c** [comment]
> Is there an expiration time for a regression to be qualified as such or should an old regression turn into regular bug work if the regression range is too old (too old needing to be defined)?

I would say the `regression` keyword should be independent from the prioritization. A regression is a regression even if it is old and less important than newer regressions.

File: policy/triage-bugzilla.md (hunk 1 above)
Comment (emceeaich) [reply policy/triage-bugzilla.md]: Adding w/link to Third-party Software Affecting Firefox

File: policy/triage-bugzilla.md (hunk 3 above)
Comment (emceeaich) [reply policy/triage-bugzilla.md]: Fixed

File: policy/triage-bugzilla.md (hunk 2 above)
Comment (emceeaich) [reply policy/triage-bugzilla.md]: Done.

Comment (emceeaich) [reply policy/triage-bugzilla.md]: * Yes
* It should be removed once the regression range is found or it's no longer needed.

File: policy/triage-bugzilla.md (hunk 4 above)
Comment (emceeaich) [reply policy/triage-bugzilla.md]: done

**Comment by emceeaich** [comment]
Regression documentation in separate page now. Merging this into it in #32.

**Pull request closed without merging**